net/mwan3/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mwan3
 PKG_VERSION:=2.6.3
-PKG_RELEASE:=2
```

Comment:
Could you please update this to PKG_RELEASE 1?
Because this is a source change an not a Makefile change.